client/payment-details/dispute-details/dispute-awaiting-response-details.tsx

```diff
@@ -53,13 +163,34 @@ const DisputeAwaitingResponseDetails: React.FC< Props > = ( {
 	const dueBy = moment.unix( dispute.evidence_details?.due_by ?? 0 );
 	const countdownDays = Math.floor( dueBy.diff( now, 'days', true ) );
 	const hasStagedEvidence = dispute.evidence_details?.has_evidence;
+	const { createErrorNotice } = useDispatch( 'core/notices' );
 	// This is a temporary restriction and can be removed once steps and actions for inquiries are implemented.
-	const showDisputeStepsAndActions = ! isInquiry( dispute );
+	const showDisputeSteps = ! isInquiry( dispute );
 
 	const onModalClose = () => {
 		setModalOpen( false );
 	};
 
+	const viewOrder = () => {
+		if ( orderUrl ) {
+			window.location.href = orderUrl;
+			return;
+		}
+
+		createErrorNotice(
+			__(
+				'Unable to view order. Order not found.',
+				'woocommerce-payments'
+			)
+		);
+	};
+
+	const disputeAcceptAction = getAcceptDisputeProps( dispute );
```

Comment:
Splitting off the key bits of content to another helper function is making this harder to understand IMO. This component is tightly coupled to `getAcceptDisputeProps`. 

I don't fully understand the different conditions needed here (e.g. how many combinations are needed). It might be possible to simplify this code by having a single component with inline conditionals, or various components hard-coded for the common scenarios, and one conditional to choose which to render.

Reply:
Hunch… if we made a component for inquiries and a component for disputes, would that make this more readable and reduce the amount of data we need to pass down as props?

Reply:
> if we made a component for inquiries and a component for disputes
This was kind of the approach I explored in #7339.

I am tackling a similar problem with the inquiry steps to resolve PR (#7292) and I think separate components might help.

I don't think we should risk pushing back the project timeline for this, however. Can we ship this with 6.6 as-is and do a small tidy-up refactor once all of the code is in place?

Reply:
Yeah totally, I don't want any of this to slow us down. We can always maintain, refactor, tidy in future :)

Reply:
Reviewing this again as I register tracks events. I really want to refactor this, `AcceptDisputeProps` is an abomination!

Reply:
I started on this refactor in #7339, which may help or may not. I will close it if not.

Reply:
Awesome! @Jinksi Do you recall where you got to / if this was feeling like positive improvement? Great to have work-in-progress to build on, though I'm happy for you to keep moving on it.

Reply:
I wrote up further in this thread that I was unsure if it was an improvement. I'll have to revisit this with fresh eyes and see.

https://github.com/Automattic/woocommerce-payments/pull/7298#discussion_r1343373487